### ISSUE TITLE
[Spotlight] Digital Collection Home Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#102]
**URL:** https://spotlight.library.tamu.edu/spotlight/alice-in-wonderland
**WCAG 2.1 AA compliant? :** No
**Violation:** 2.4.5 Multiple Ways (AA)
**Issues:**  No way to navigate back to the main spotlight home page
**Notes:**

**Comment by qtamu** [comment]
Each collection has its own home. Would need some design questions answered on how and where we want this navigation link to be when there is an existing home button.
